zipline/assets/assets.py

```diff
@@ -823,17 +928,34 @@ def lookup_symbol(self, symbol, as_of_date, fuzzy=False):
             Raised when no ``as_of_date`` is given and more than one equity
             has held ``symbol``. This is also raised when ``fuzzy=True`` and
             there are multiple candidates for the given ``symbol`` on the
-            ``as_of_date``.
+            ``as_of_date``. Also raised when no ``country_code`` is given and
+            the symbol is ambiguous across multiple countries.
         """
         if symbol is None:
             raise TypeError("Cannot lookup asset for symbol of None for "
                             "as of date %s." % as_of_date)
 
         if fuzzy:
-            return self._lookup_symbol_fuzzy(symbol, as_of_date)
-        return self._lookup_symbol_strict(symbol, as_of_date)
+            f = self._lookup_symbol_fuzzy
+            mapping = self._which_fuzzy_symbol_ownership_map(country_code)
+        else:
+            f = self._lookup_symbol_strict
+            mapping = self._which_symbol_ownership_map(country_code)
+
+        if mapping is None:
```

Comment:
it seems like that should raise a consistent error that says, "I don't know about this symbol country pair". I am imagining a user who has multi-country symbols to map, and the assets db only knows about a subset of the symbols. If we add a new error, they now need to care about these different failed lookups instead of handling it the same way.